poly-commitment/src/commitment.rs

```diff
@@ -332,19 +333,29 @@ impl<C: AffineRepr> PolyComm<C> {
         let all_scalars: Vec<_> = elm.iter().map(|s| s.into_bigint()).collect();
 
         let elems_size = Iterator::max(com.iter().map(|c| c.chunks.len())).unwrap();
```

Comment:
The code LGTM in terms of functionality.
However, I do not see any unit test and bench for this particular function (only some using `msm` directly).
I think it would be nice to have to compare before and after, and for regression (bencharmking) tests later (as I feel the performance team will have later?).